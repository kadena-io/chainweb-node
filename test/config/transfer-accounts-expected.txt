[TxLog {_txDomain = "USER_test1_accounts", _txKey = "Acct1", _txValue = Object (fromList [("amount",Object (fromList [("_P_decm",Number -100.0),("_P_decp",Number 2.0)])),("data",Object (fromList [("_P_val",Object (fromList [("transfer-to",String "Acct2")]))])),("balance",Object (fromList [("_P_decm",Number 9.99999e7),("_P_decp",Number 2.0)]))])}]