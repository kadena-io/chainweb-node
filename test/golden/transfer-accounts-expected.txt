- output: SWXWAANubWRDjkJaWluDxsG37jA9OkKP-WYICdx33Uw
  cmd: coinbase
- output: fwUxLBrxLcx0lV4SnFULKaBtwv_6NkLsHDhVIQK-UZ8
  cmd:
    tag: Code
    contents: (test1.transfer "Acct1" "Acct2" 1.00)
